pages/requests/new/index.js

```diff
@@ -1,6 +1,69 @@
-// TODO(alishaevn): redo this page when the time comes
+import React, { useState } from 'react'
+import {
+  Title,
+} from 'webstore-component-library'
+
 // TODO(alishaevn): trying to access this page without being signed in should redirect to the login page
 
-const NewBlankRequest = () => <h1>This is a new blank request.</h1>
+const NewBlankRequest = () => {
+  /*
+  based on the json response when getting a request, lets assume that our post request to create a new request may accept a body with the following properties:
+  {
+    description,
+    billing_same_as_shipping,
+    proposed_deadline,
+    shipping_address,
+    billing_address
+  }
+  */
+
+  // have a function here that accepts our data and takes it to the post api call
+  // e.g. postApiCall(shippingAddress, billingAddress, billingSameAsShipping, description, proposedDeadline)
+
+  // set up some local state
+  const [shippingAddress, setShippingAddress] = useState(null)
+  const [billingAddress, setBillingAddress] = useState(null)
+  const [billingSameAsShipping, setBillingSameAsShipping] = useState(null)
+  const [description, setDescription] = useState(null)
+  const [proposedDeadline, setProposedDeadline] = useState(null)
+
+  // set up a function that handles the data. the below won't work because we need an onChange for the fields
+  // or do we just use a react bootstrap form for this page??
```

Comment:
@summer-cook your thoughts on the approach on this line?

Reply:
we should definitely use a react bootstrap form- i think that might make it easier

Reply:
summer and I continued this convo verbally. we will instead build out this page as a component in the library since it's all one big form.

then we will render that component on this page.